symbulate/distributions.py

```diff
@@ -341,14 +344,30 @@ class Normal(Distribution):
     """
```

Comment:
FYI (no change needed): this docstring is actually wrong at this point and will need to be modified sometime down the road.